packages/oapp-evm/foundry.toml

```diff
@@ -0,0 +1,20 @@
+[profile.default]
+solc = '0.8.22'
+verbosity = 3
+src = "contracts"
+test = "test"
+out = "artifacts"
+cache_path = "cache"
+optimizer = true
+optimizer_runs = 20_000
```

Comment:
You can speed up the build by removing this or setting it to like 200 - we are not deploying these directly so having the optimizer here does not matter at all